package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "timestamp-microservice",
+  "version": "1.0.0",
+  "main": "timestamp.js",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "ejs": "^2.6.1",
+    "express": "^4.16.4",
+    "nodemon": "^1.18.10"
```

Comment:
why do you need
1. Nodemon - this is a command line utility and should not be bundled with project
2. ejs - you are not rendering anything
3. body parser - i dont see you using it.